packages/shared/src/components/tooltips/ChangelogTooltip.tsx

```diff
@@ -99,138 +90,122 @@ function ChangelogTooltip<TRef extends HTMLElement>({
+            src={post.image}
+            fallbackSrc={cloudinary.post.imageCoverPlaceholder}
+            loading="lazy"
+            data-testid="changelogImage"
+          />
+          <h3
+            className="mt-3 font-normal text-theme-label-primary typo-title3"
+            data-testid="changelogTitle"
           >
-            <header className="flex flex-1 items-center py-3 px-4 border-b border-theme-divider-tertiary">
-              <h3
-                className="font-bold typo-title3 text-theme-label-primary"
-                data-testid="changelogNewReleaseTag"
-              >
-                New release
-              </h3>
-              <ModalClose
-                className="right-4"
-                onClick={onModalCloseClick}
-                data-testid="changelogModalClose"
-                buttonSize={ButtonSize.XSmall}
+            {post.title}
+          </h3>
+          <time
+            className="text-theme-label-quaternary typo-callout"
+            dateTime={post.createdAt}
+            data-testid="changelogDate"
+          >
+            {postDateFormat(post.createdAt)}
+          </time>
+          {!!post.summary && (
+            <div
+              className="mt-2 w text-theme-label-tertiary typo-callout"
+              data-testid="changelogSummary"
+            >
+              {post.summary}
+            </div>
+          )}
+          <div className="flex mt-4 font-bold w text-theme-color-salt typo-footnote">
+            <Button
+              id={`post-${post.id}-upvote-btn`}
+              icon={
+                <UpvoteIcon
+                  secondary={post?.userState?.vote === UserPostVote.Up}
+                />
+              }
+              pressed={post?.userState?.vote === UserPostVote.Up}
+              onClick={onToggleUpvote}
+              buttonSize={ButtonSize.Small}
+              className="btn-tertiary-avocado bg-theme-bg-tertiary"
+              data-testid="changelogUpvotesCounter"
+            >
+              <InteractionCounter
+                value={post.numUpvotes > 0 && post.numUpvotes}
               />
-            </header>
-            <section className="flex flex-col flex-1 p-5 h-full shrink max-h-full">
-              <Image
-                className="object-cover rounded-lg w-[207px] h-[108px]"
-                alt="Post Cover image"
-                src={post.image}
-                fallbackSrc={cloudinary.post.imageCoverPlaceholder}
-                loading="lazy"
-                data-testid="changelogImage"
+            </Button>
+            <Button
+              id={`post-${post.id}-comment-btn`}
+              icon={<CommentIcon secondary={post.commented} />}
+              pressed={post.commented}
+              tag="a"
+              href={post.commentsPermalink}
+              onClick={dismissChangelog}
+              buttonSize={ButtonSize.Small}
+              className="btn-tertiary-blueCheese bg-theme-bg-tertiary"
```

Comment:
Here as well the bg is not needed, and think it should be quarternary button.

Reply:
Same comment as on the upvote button -> https://github.com/dailydotdev/apps/pull/2274#discussion_r1366779234